internal/bazeldnf.bzl

```diff
@@ -52,7 +52,7 @@ _bazeldnf = rule(
         "libs": attr.string_list(),
         "tar": attr.label(allow_single_file = True),
         "_bazeldnf": attr.label(
-            default = "@bazeldnf//cmd:cmd",
+            default = "@bazeldnf//cmd:prebuilt",
```

Comment:
Should we have here:


```python
default = select ({
  "@platforms//cpu:x86_64": "@bazeldnf//cmd:prebuilt",
  "//conditions:default": "@bazeldnf//cmd:cmd",
})
```

not sure if this would be the exact correct select statement, or if it can be written here like this.

Reply:
Also we probably need to have this select the built-in version if we directly develop on it or run CI. Do you have experience on how other projects do that? I think some, like protoc related projects, have something like this in place ...

Reply:
oh my, now I see that the select is in the prebuilt :D